packages/ui5-info/src/ui5-version-info.ts

```diff
@@ -148,14 +148,9 @@ async function parseUI5VersionsOverview(): Promise<UI5VersionOverview[]> {
         );
         versions = ui5VersionFallbacks;
     }
-    result = versions.map((ver: any) => {
+    result = versions.map((ver: any): UI5VersionOverview | undefined => {
         const parsedVersion = coerce(ver.version)?.version;
-        if (parsedVersion !== undefined) {
-            return {
-                version: parsedVersion,
-                support: ver.support
-            };
-        }
+        return parsedVersion !== undefined ? { version: parsedVersion, support: ver.support } : undefined;
```

Comment:
just a code smell fix: add return type to arrow function and make sure it has an ending return